### PR TITLE
Feat:Input and navbar added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Hi guys
Please review my input and navbar component.

### CODE REVIEW

**Comment by palak-dhingra** [comment]
looks good to merge

**Review by arulk8 — COMMENTED** [review]
Add Margin-left: auto; to your .right-nav class. login,cart and Wishlist icon are not aligning properly

File: Styles/nav.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+.left-nav{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width:700px;
+    padding-right: 12em;
+}
+.right-nav{
+    display: flex;
```
Comment (arulk8) [inline Styles/nav.css:20]: margin-left: auto

**Review by arulk8 — COMMENTED** [review]
Looks good to me.

**Review by sumitch29 — APPROVED** [review]
center 16 number inside icon badge inside navbar. And link component page to sidebar, while clicking input. Otherwise good to go.

**Pull request closed without merging**